src/apps/fee-list/stackable-fees/fee-info.tsx

```diff
@@ -50,7 +51,7 @@ const FeeInfo: FC<FeeInfoProps> = ({
         </div>
         <div />
         <p className="text-small-caption color-secondary-gray">
-          {materialItemNumber}
+          {`${t("feeLitsMaterialNumberText")} ${materialItemNumber}`}
```

Comment:
I think you should ude the placeholder functionality of the t function.
Example: https://github.com/danskernesdigitalebibliotek/dpl-react/blob/80678f8839dbb8d411014ac1495c874a30293a1d/src/apps/loan-list/materials/selectable-material/selectable-material.tsx#L141-L143